src/Shaders/ShadersInclude/morphTargetsVertex.fx

```diff
@@ -3,5 +3,10 @@
 	
 	#ifdef MORPHTARGETS_NORMAL
 	normalUpdated += (normal{X} - normal) * morphTargetInfluences[{X}];
-	#endif	
+	#endif
+
+	#ifdef MORPHTARGETS_TANGENT
+	tangentTemp = tangentUpdated.xyz + ((tangent{X} - tangent.xyz) * morphTargetInfluences[{X}]);

```

Comment:
Because if we have have 4 targets the final formula:

tangentTemp = 4 * tangentUpdated.xyz + sum(((tangent{X} - tangent.xyz) * morphTargetInfluences[{X}]));